cmd/cluster/health.go

```diff
@@ -192,17 +228,31 @@ func ocmDescribe(clusterID string) ([]string, string, int, error) {
 	cluster := response.Body()
 	cloudProvider := cluster.CloudProvider().ID()
 	cloudProviderMessage := strings.ToUpper(cloudProvider)
+	clusterInfo.Name = cluster.Name()
 
 	fmt.Printf("\nCluster %s - %s\n", cluster.Name(), cluster.ID())
 	fmt.Printf("\nCloud provider: %s\n", cloudProviderMessage)
 
 	if cloudProvider != "aws" {
-		return nil, "", 0, fmt.Errorf("This command is only supported for AWS clusters. The command is not supported for %s clusters.", cloudProviderMessage)
-
+		clusterInfo.AZs = nil
+		clusterInfo.Name = ""
```

Comment:
You don't need to set all these (238-243). Go will fill them in with the respective type's default value.

Reply:
Thanks for the tip, @fahlmant!